projects/envoy/find_corpus.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/python
+import os, sys, re
```

Comment:
Should ideally use Google Python style here; https://github.com/google/styleguide/blob/gh-pages/pyguide.md. For the spacing, I would write:
```
#!/usr/bin/python

import os
import re
import sys

<rest of code>
```